src/groovy/au/org/emii/portal/HostVerifier.groovy

```diff
@@ -12,6 +12,9 @@ class HostVerifier {
     def allowedHosts = null
 
     def allowedHost(address) {
+        if (grailsApplication.config.allowAnyHost) {
+            return true
+        }
 
         initializeAllowedHostsIfNeeded()
 
```

Comment:
There may be value in placing this new code latter, after the test for an empty address? You could also then remove a return?
```
if (allowedHosts[host] || grailsApplication.config.allowAnyHost) {
            return true
        }
```